lib/css_parser/rule_set.rb

```diff
@@ -22,6 +22,30 @@ class RuleSet
       ['border-width', %w[border-top-width border-right-width border-bottom-width border-left-width]]
     ].freeze
 
+    class DeclarationValue
+      attr_reader :value
+      attr_accessor :important, :order
+
+      def initialize(value, important: nil, order: 0)
+        self.value = value
+        @important = important.nil? ? !value.match(CssParser::IMPORTANT_IN_PROPERTY_RX).nil? : important
```

Comment:
>this seems to do a lot of work for cases were we don't need it
Made it ternary operator, so we don't run regexp if it's not needed (i. e. `important:` is passed). 
>important can just be true/false no need to dead with nils ?
We should somehow check if `important:` was passed or we should calculate importance from the value, `.nil?` looks like appropriate solution.

Reply:
can we change match -> match?

Reply:
`match?` is not available on ruby-2.4.

Reply:
we still do the important parsing twice ...
so I think we assume `value=` cannot change important which is a little strange but ok
then we should do something like this:
```
if important.nil? # user did not tell us, so we find out
  value = value.to_s.dup
  important = !!value.sub!(CssParser::IMPORTANT_IN_PROPERTY_RX, '')
end
```

... or ideally do that in `value=` by changing it to `set_value(value, important: nil)` since it's kinda more logical that setting the value can influence important ?

Reply:
... or even `set_expression` since it does not really set only the value

Reply:
can bump min ruby to 2.5 since 2.4 is eol anyway 🤷